requirements.txt

```diff
@@ -6,16 +6,15 @@ munch>=2.1
 networkx>=2.0
 ostrichlib>=0.1
 requests>=2.18.0
-# seems google-cloud-storage needs six without declaring it.
```

Comment:
why remove this comment? is it not true?

Reply:
that's actually not true I think. 
google-cloud-storage uses it through the api-core which does declare this:
https://github.com/googleapis/python-api-core/blob/master/setup.py#L37

and we did have six installed just with an older package, I actually not sure how package versions is picked up on travis.
I'll leave a comment saying that